fee/gas.go

```diff
@@ -10,6 +11,33 @@ const (
 	payloadGasSize = uint64(5)
 )
 
+func IsUseTxGas(version int32) bool {
+	return !IsZeroFee() && version >= 2
+}
+
+func IsVmGasSystem(version int32, isQuery bool) bool {
+	return IsUseTxGas(version) && !isQuery
+}
+
+func IsReceiptGasUsed(version int32, isGovernance bool) bool {
+	return IsUseTxGas(version) && !isGovernance
+}
```

Comment:
I also do not see any reason for this function to exist

Reply:
it resolved in https://github.com/aergoio/aergo/commit/cc4ffbf860757323fe485c62d925fa26dc0a1248